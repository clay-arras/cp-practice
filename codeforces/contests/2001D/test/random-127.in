1
20
5 8 12 24 14 21 19 6 22 22 17 17 23 25 16 10 17 11 4 1
