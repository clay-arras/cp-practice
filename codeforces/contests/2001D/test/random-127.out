17
5 8 12 24 14 21 19 6 22 17 23 25 16 10 11 4 1
